src/lib/isMobilePhone.js

```diff
@@ -41,7 +41,7 @@ const phones = {
   'en-BS': /^(\+?1[-\s]?|0)?\(?242\)?[-\s]?\d{3}[-\s]?\d{4}$/,
   'en-GB': /^(\+?44|0)7\d{9}$/,
   'en-GG': /^(\+?44|0)1481\d{6}$/,
-  'en-GH': /^(\+233|0)(20|50|24|54|27|57|26|56|23|28|55|59)\d{7}$/,
+  'en-GH': /^(\+233|0)(20|50|24|54|27|57|26|56|23|28|55|59|53|25)\d{7}$/,
```

Comment:
According to https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000520009PDFE.pdf only 2540, 256 and 257 are valid.

53 is fine.